### PR TITLE
Move a number of ui only updates into UiContext tick

### PR DESCRIPTION
**Pull request by duncanspumpkin**
This was just to clear up where the window dispatch update was being called but thought i might as well also move over these other Ui only items.

### CODE REVIEW

File: src/openrct2-ui/UiContext.cpp [hunk 1]
```diff
@@ -140,6 +140,28 @@ class UiContext final : public IUiContext
     void Tick() override
     {
         _inGameConsole.Update();
+
+        // the flickering frequency is reduced by 4, compared to the original
+        // it was done due to inability to reproduce original frequency
+        // and decision that the original one looks too fast
+        if (gCurrentRealTimeTicks % 4 == 0)
```
Comment (Gymnasiast) [inline src/openrct2-ui/UiContext.cpp:147]: This file needs to import `Game.h`  (where `gCurrentRealTimeTicks` is declared).

Comment (duncanspumpkin) [reply src/openrct2-ui/UiContext.cpp:147]: I was thinking about it more and i think I'll actually move this to the map window as part of the update function as I'm pretty sure that's the only user

**Review by Gymnasiast — APPROVED** [review]
Code LGTM, needs testing.

**Comment by Gymnasiast** [comment]
Testing revealed no bugs.

The framerate when opening the Map window is abysmal, but that also happens on develop.

**Pull request merged**